src/main/java/com/mixpanel/android/mpmetrics/MixpanelFCMMessagingService.java

```diff
@@ -170,8 +136,8 @@ public static void showPushNotification(Context context, Intent messageIntent) {
 
         final ResourceIds drawableIds = new ResourceReader.Drawables(resourcePackage, context);
         final Context applicationContext = context.getApplicationContext();
-        final Notification notification = buildNotification(applicationContext, messageIntent, drawableIds);
-
+        MixpanelPushNotification mixpanelPushNotification = new MixpanelPushNotification(applicationContext, drawableIds);
```

Comment:
we should keep the implementation so developers can override how the notification is built if they want to add new features or modify settings (that's why `makeNotificationIf...` methods were protected if devs are overriding `MixpanelFCMMessagingService`).

We need a way to let developers override the notification before its posted to the notificationmanager